openapi/components/schemas/TagUntag/TagUntagRule.yaml

```
@@ -0,0 +1,94 @@
+type: object
+description: Tag / Untag rule.
```

suggestion: I think we should avoid using `/` slash in descriptions

```suggestion
description: Tag or untag rules.
```
or 

```suggestion
description: Tag or untag a rule.
```